test/integration/apiserver/discovery/discovery_test.go

```diff
@@ -97,6 +98,28 @@ var (
 		},
 	}
 
+	basicTestGroupWithFixup = apidiscoveryv2beta1.APIGroupDiscovery{
```

Comment:
Just want to confirm, this is specifically testing that the aggregator will apply the fix up even if the underlying apiservice doesn't populate the empty GVK? (as opposed to directly setting `basicTestGroupWithFIxup` on L222)